library/src/morpheus-master/bootstrap-sass-3.3.7/assets/stylesheets/bootstrap/_variables.scss

```diff
@@ -490,8 +490,8 @@ $pager-disabled-color:                 $pagination-disabled-color !default;
 //##
 
 $jumbotron-padding:              30px !default;
-$jumbotron-color:                inherit !default;
-$jumbotron-bg:                   $gray-lighter !default;
+$jumbotron-color:                var(--jumbotron-color, inherit);
+$jumbotron-bg:                   var(--jumbotron-bg, #{$gray-lighter});
 $jumbotron-heading-color:        inherit !default;
```

Comment:
I think changes to Bootstrap defaults should be made in: library/src/morpheus-master/sass/base/_bootstrap-defaults.scss